conf/defaults.yaml

```diff
@@ -231,16 +228,13 @@ topology.multilang.serializer: "org.apache.storm.multilang.JsonSerializer"
 topology.shellbolt.max.pending: 100
 topology.skip.missing.kryo.registrations: false
 topology.max.task.parallelism: null
-topology.max.spout.pending: null
+topology.max.spout.pending: null  # TODO: We dont need this any more
```

Comment:
I guess it utilizes blocking queue which looks natural for the definition of back pressure. If then we need to be sure there's no livelock and deadlock. AFAIK, Storm used blocking queue before, and ended up adding overflow buffer because of deadlock (dining philosopher).